OrbitQt/TopDownViewItemModel.cpp

```diff
@@ -79,13 +82,27 @@ QVariant TopDownViewItemModel::GetEditRoleData(const QModelIndex& index) const {
         return static_cast<qulonglong>(function_item->GetExclusiveSampleCount());
       case kOfParent:
         return static_cast<qulonglong>(function_item->GetPercentOfParent());
+      case kModule:
+        return QString::fromStdString(function_item->GetModuleName());
       case kFunctionAddress:
         return static_cast<qulonglong>(function_item->function_absolute_address());
     }
   }
   return QVariant();
 }
 
+QVariant TopDownViewItemModel::GetToolTipRoleData(const QModelIndex& index) const {
+  auto* item = static_cast<TopDownNode*>(index.internalPointer());
```

Comment:
Nit: maybe a `CHECK(index.isValid())`? Makes it easier to understand the function when it's seen on its own.

Reply:
Good point, done.